integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSession.scala

```diff
@@ -110,6 +110,14 @@ class CarbonSession(@transient val sc: SparkContext,
     )
   }
 
+  /**
+   * Return true if the specified sql statement will hit the datamap
+   */
+  def isDataMapHit(sqlStatement: String, dataMapName: String): Boolean = {
```

Comment:
Is this method only for tests? If so, better to limit its usage, such as adding annotation 'forTests' or other restictions

Reply:
fixed